Esp8266/Esp8266.ino

```diff
@@ -1,86 +1,116 @@
-#include "event.pb.h"
-
-#include <pb_common.h>
-#include <pb.h>
-#include <pb_encode.h>
-#include <pb_decode.h>
+#include "pb_common.h"
+#include "pb.h"
+#include "pb_encode.h"
+#include "pb_decode.h"
+#include "Esp8266_Interface.pb.h"
 
 #include <ESP8266WiFi.h>
 
-const char* ssid     = "onhub";
-const char* password = "ekchotaghar";
-const char* addr     = "10.0.1.181";
-const uint16_t port  = 10100;
+const char* ssid     = "SSID";
```

Comment:
Here is an idea, why not have the ssid and pass in a file that the program reads from? that way, we can make this program more agile and scalable.

Reply:
That's what I was going to do next. I was going to make a file called SSID_PASS.h and #include it, and then add a comment above this line describing its contents (I would also put it in gitignore).

Reply:
Woody.